relayer/gasless.py

```
@@ -130,51 +178,76 @@ def main():
     web3 = Web3(Web3.HTTPProvider(args.rpc))
     chain_id = web3.eth.chain_id
 
-    # permit(web3, args.private_key, args.sender_private_key, args.input_token, args.input_amount, args.origin_settler)
+    account = Account.from_key(args.private_key)
+
+    deadline, v, r, s = permit(web3, account, args.input_token, args.input_amount, args.origin_settler)
 
     contract = web3.eth.contract(args.origin_settler, abi=json.load(open("abi.json"))["abi"])
```

Ruff

Use a context manager for opening files

(SIM115)